content/get-started/get-started.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Getting Started
+seo-title: Getting Started
+---
+
+Iron Fish is cryptocurrency that allows for safe, secure, and private transactions.
+
+This guide is intended for crypto novices to understand what Iron Fish is and how they can get started using it. If you want to skip directly to installing and running Iron Fish, you can [install a node](/use/get-started/installation) now.
+
+The currency symbol for Iron Fish is $IRON. This is compared to United States Dollar (USD), Chinese Yuan (CNY), or Russian Ruble (RUB).
+
+### Why use $IRON?
+
+Currencies like $IRON promote decentralization by creating a global ledger that no single person can modify. When you run Iron Fish, you help keep this ledger secure by checking that transactions are following the balances in the ledger. By running Iron Fish, you help keep your and your communities coins secure.
+
+The advantage of $IRON is that it's fundamentally private. Other people cannot see the amount sent, or the recipient without your permission. That's thanks to technology called Zero Knowledge Proofs. This is different from Bit Coin which is fundamentally public.
+
+### How can I use $IRON?
+
+There are three way Fish network. Choosing which one is the right solution for you depends on your goals and technical skill.
+
+#### Wallets
+
+Wallets are a simple easy to use way to send and recieve $IRON.
+
+They allow you to send and recieve coins. When you use a wallet, other Iron Fish users will validate your transactions before they are added to the ledger.
```

Comment:
```suggestion
Wallets are a simple easy to use way to send and recieve $IRON and other assets. When you use a wallet, other Iron Fish users will validate your transactions before they are added to the ledger.
```

Reply:
I don't think we should mention assets yet here. People will say, wait I thought I send coins, what is an asset?